### DISCUSSION TITLE
A question about installed packages

### DISCUSSION THREAD

**Discussion by dev02iggy** [nuxt/nuxt#16974] [General]
I would like to understand why the packages are in the part of development dependencies ("devDependencies"), instead of being in the production dependencies.
This would help me, because I'm installing normally in the production and I don't know if this will cause problems in the future.

Note: I don't speak English very well.

**Comment by danielroe** [comment]
See https://github.com/nuxt/framework/discussions/5880.

**Reply by dev02iggy** [reply]
thanks!
